src/lib/components/forms/form/form-components/slider/utils/getNumberOfFractionalDigits.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { getNumberOfFractionalDigits } from './getNumberOfFractionalDigits';
+
+describe('getNumberOfFractionalDigits', () => {
+    it('should return 0 if the precision is undefined', () => {
+        expect(getNumberOfFractionalDigits(undefined)).toEqual(0);
+    });
+
+    it('should return the number of fractional digits if an interval is set to 0.1', () => {
+        expect(getNumberOfFractionalDigits(0.1)).toEqual(1);
```

Comment:
Gerne noch ein 2 weitere Fälle testen :) Z.B.
```suggestion
        expect(getNumberOfFractionalDigits(1223)).toEqual(0);
        expect(getNumberOfFractionalDigits(0.1)).toEqual(1);
        expect(getNumberOfFractionalDigits(12.132)).toEqual(3);
```